api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,115 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /timelines:
+    post:
+      tags:
+        - timelines-api
+      description: |
+        Creates a new timeline for an event type and makes it active.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      parameters:
+        - name: timeline_request
+          in: body
+          schema:
+            description: Event type and storage id to be used for timeline creation.
+            type: object
+            properties:
+              event_type:
+                type: string
+              storage_id:
+                type: string
+          required: true
+      responses:
+        '201':
+          description: New timeline is created and in use.
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /timelines/{id}#1:
+    delete:
+      tags:
+        - timelines-api
+      description: |
+        Deletes a timeline by its id. Method call is quite dangerous, because it will remove any timeline.
```

Comment:
I do not agree with any timeline. It should delete only first timeline if it is last at the same time.